.travis.yml

```diff
@@ -3,7 +3,9 @@ env:
 addons:
   ssh_known_hosts: 157.230.182.92
 script:
-- docker build .
+- docker build -t test-cg-admin-app .
+- docker run -e CI=true -d --name test test-cg-admin-app
```

Comment:
Can we leave app out of the name it’s redundant

Reply:
It really does not matter what we put there because it is just is used keep track of the local image name. It will not be used anywhere else.